src/gui/errorspopup.cpp

```diff
@@ -45,7 +45,7 @@ static const int shadowBlurRadius = 20;
 static const int menuOffsetX = -30;
 static const int menuOffsetY = 10;
 
-const std::string actionTypeProperty = "actionType";
+const std::string ActionTypeProperty = "actionType";
```

Comment:
Do not start name of variables with upper case

Reply:
Done.